src/main.c

```diff
@@ -2752,7 +2812,7 @@ se_debug_tool_desc_t nds_debug_tools[]={
   {ICON_FK_PENCIL_SQUARE_O " 9", ICON_FK_PENCIL_SQUARE_O " ARM9 Memory",nds9_mem_debugger},
   {ICON_FK_INFO_CIRCLE, ICON_FK_INFO_CIRCLE " NDS IO",nds_io_debugger},
 
-  {ICON_FK_AREA_CHART, ICON_FK_AREA_CHART " Emulator Stats",se_draw_emu_stats, .allow_hardcore=true},
+  {ICON_FK_AREA_CHART, ICON_FK_AREA_CHART " Emulator Stats",se_draw_emu_stats, .allow_hardcore=true}, // TODO: probably should not allow hardcore. ig Pokemon Black unlocks 6 achievements on boot
```

Comment:
Same with this comment?

Reply:
The pokemon black thing was a bug in how I handled read from RAM, it has been since fixed, I forgot to remove this comment